ContainerToolsForWindows/Public/AllToolsUtilities.psm1

```diff
@@ -1,16 +1,15 @@
-$ModuleParentPath = Split-Path -Parent $PSScriptRoot
-Import-Module -Name "$ModuleParentPath\Private\SetupUtilities.psm1" -Force
+﻿$ModuleParentPath = Split-Path -Parent $PSScriptRoot
+Import-Module -Name "$ModuleParentPath\Private\CommonToolUtilities.psm1" -Force
```

Comment:
Nit: does powershell not allow 2 different files named utilities e.g. `Private\Utilities` and `Public\Utilitites`

Reply:
This will cause module name conflicts given that Private\Utilities.psm1 and Public\Utilitites.psm1 are both modules.
[Manage name conflicts](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_modules?view=powershell-7.4#manage-name-conflicts)